tests/unit/backend/layers/api/test_curation_api.py

```diff
@@ -1023,17 +1023,15 @@ def test_get_collection_version_ok(self):
                     "collection_version_id": f"{first_version.version_id.id}",
                     "cell_count": 10,
                     "primary_cell_count": 5,
-                    "cell_type": [{"label": "test_cell_type_label", "ontology_term_id": "test_cell_type_term_id"}],
+                    "cell_type": [{"label": "unknown", "ontology_term_id": "unknown"}],
```

Comment:
Are these`unknown`s from `OntologyParser`?

Reply:
no, its just that the parser will throw exceptions if we pass in invalid term IDs like "test_term_id" but it is built to accept "unknown" as an exception case